internal/pkg/manifest/job.go

```diff
@@ -60,9 +60,10 @@ type JobFailureHandlerConfig struct {
 // ScheduledJobProps contains properties for creating a new scheduled job manifest.
 type ScheduledJobProps struct {
 	*WorkloadProps
-	Schedule string
-	Timeout  string
-	Retries  int
+	Schedule    string
+	Timeout     string
+	HealthCheck *ContainerHealthCheck // Optional healthcheck configuration.
```

Comment:
OK so the only time we deem container healthchecks to be useful for scheduled jobs is if they have sidecars and want to use container dependencies. Otherwise, I can't think of a use for it for ephemeral tasks 🤔